playbooks/test.yml

```diff
@@ -49,7 +49,7 @@
         broker_dao_type: etcd
 
     - name: Wait for clusterservicebroker to become available
-      command: kubectl get clusterservicebrokers automation-broker -o go-template={%raw%}'{{ range .status.conditions }}{{ if eq .type "Ready" }}{{ .status }}{{ end }}{{ end }}'{%endraw%}
+      command: kubectl get clusterservicebrokers ansible-service-broker -o go-template={%raw%}'{{ range .status.conditions }}{{ if eq .type "Ready" }}{{ .status }}{{ end }}{{ end }}'{%endraw%}
```

Comment:
Same comment as above, just use `cluster_service_broker_name`, but still default to ansible-service-broker